1-estrategia/README.md

```diff
@@ -1,23 +1,162 @@
+- Interactividad: todo contenido publicado en la web podrá ser valorado, comentado y compartido por el usuario. Los botones para valorar/comentar estarán situados al final de toda publicación.
+- Accesibilidad a las redes sociales: los botones que enlazan al usuario con nuestras redes sociales estarán situados en el homepage junto al nombre del portal.
 
 ### Indicadores de éxito
+- Los usuarios tardan menos de 10 segundos en entrar a la sección a la que quieran acceder.
+- El 50% de los lectores que acceden a contenidos valoran o comentan.
+- El 20% de los usuarios comparten contenidos en sus redes sociales.
+- El 80% de los usuarios que hacen uso del buscador encuentran el contenido.
+- El 50% de las visitas que reciben nuestras redes sociales llegan desde los botones de la página web.
+- Ningún contenido estará sin enlazar con otros contenidos.
 
 ## Necesidades del usuario
 
 ### Investigación de usuarios
 
+En este apartado se investigará a los usuarios para saber cuales son sus necesidades, para así diseñar la página web en base a las necesidades detectadas. Para ello, hay que tener claro cuál es el público objetivo al que se dirigirá la página web. En nuestro caso, el público objetivo son las personas aficionadas a la música de entre 16 y 40 años. Una vez identificado el público, el segundo paso será hacer una investigación sobre las necesidades de los usuarios.
+
+Existen muchas técnicas para identificar estas necesidades, pero en este caso, hemos optado por hacer una encuesta. El objetivo de esta encuesta es lograr identificar los gustos musicales y el hábito de consumo de música en Internet de los usuarios. El público elegido para rellenar la encuesta son algunas personas de nuestro entorno de entre 16 y 40 años con afición a la música (público objetivo).  
+
+La encuesta que hemos diseñado es breve y fácil de rellenar. Cuenta con 11 preguntas cerradas sobre el tema de la música para saber cuáles son los intereses de los encuestados y cómo consumen la música en Internet. También nos ha parecido relevante preguntar la edad a los encuestados porque creemos que nos dará una idea general de los gustos y los hábitos de consumo de música que tienen las personas de dicha edad. Hemos descartado preguntar el sexo y la localidad porque creemos que estos datos no son relevantes para esta investigación.
+
+**ENCUESTA: CONSUMO DE LA MÚSICA EN INTERNET**
+
+¡Hola! Queremos diseñar una página web musical para ofrecer información de todo tipo acerca de este tema. Para ello, estamos realizando una investigación sobre el consumo de la música en Internet y necesitamos tu ayuda. Por favor, rellena este formulario cuando puedas. Muchas gracias.
```

Comment:
Tanto en vuestro planteamiento como aquí, en la encuesta que presentáis a los usuarios, mezcláis el consumo de **información acerca de la** música, con el consumo de música, que no son la misma cosa. Cuidado con la manera de redactar.